gameObjects.py

```diff
@@ -149,16 +151,16 @@ def calculate_location(self, velocity):
 
     # Initialize the move but don't do it twice
     def start_move(self):
-        if self.move_start_time is 0:
+        if self.move_start_time == 0:

```

Comment:
Do you know why this change is suggested?  I noticed when comparing to `None` it suggested the opposite.  I'm guess it has something to do with implicit type conversion?

Reply:
Identity is not the same thing as equality in Python so use ==/!= to compare str, bytes, and int literals. In Python >= 3.8, these instances will raise __SyntaxWarnings__ so it is best to fix them now.  https://docs.python.org/3.8/whatsnew/3.8.html#porting-to-python-3-8

% __python__
```
>>> present = "pres"
>>> present += "ent"
>>> present == "present"
True
>>> present is "present"
False
>>> 0 == 0.0
True
>>> 0 is 0.0
False
```

Reply:
None, True, False are singletons in Python so using __is__ is what you want.

Reply:
Thank you kindly.